integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -88,20 +97,20 @@ void testLoadCatalogs() {
   void testCreateAndLoadSchema() {
     String testDatabaseName = "t_create1";
     dropDatabaseIfExists(testDatabaseName);
-    sql("CREATE DATABASE " + testDatabaseName);
+    sql("CREATE DATABASE " + testDatabaseName + " WITH DBPROPERTIES (ID=001);");
```

Comment:
does Iceberg add some db properties by default?

Reply:
yes, it add owner as the default property.

Reply:
org.apache.spark.sql.connector.DataSourceV2SQLSuite
```
  test ("11") {
    sql("create database if not exists testcat.ns1.ns3")
    sql("desc database testcat.ns1.ns3").show(false)
  }
```

```
+--------------+----------+
|info_name     |info_value|
+--------------+----------+
|Namespace Name|ns1.ns3   |
|Owner         |test        |
+--------------+----------+
```

Reply:
![image](https://github.com/datastrato/gravitino/assets/94670132/543599aa-ce11-43c9-844a-fefae3544b52)

Reply:
ok